ToolsForMaintainersOfTheProjectTemplate/Scintilla_iface_synchronizer/Face.py

```diff
@@ -50,6 +50,10 @@ def ReadFromFile(self, name):
 		currentCategory = ""
 		currentComment = []
 		currentCommentFinished = 0
+		import os, os.path, inspect
+		if '__file__' not in locals(): 
+			__file__ = inspect.getframeinfo(inspect.currentframe())[0]
```

Comment:
set cwd to script path for traversing relative file paths correctly such as ../../ etc